CONTRIBUTING.md

```diff
@@ -215,16 +215,19 @@ flutter test test/ --update-goldens
 
 ## Manual Testing
 
-To run master version of DevTools with all experimental features enabled:
+If you do not have environment configured, but want to try newest DevTools with all experimental features enabled:
 
-1. Start DevTools
+1. Configure Dart or Flutter.
+
+2. Start DevTools:
 ```
 git clone git@github.com:flutter/devtools.git
+sudo ./devtools/tool/update_flutter_sdk.sh
 cd devtools/packages/devtools_app
-flutter run -d chrome --dart-define=enable_experiments=true
+../../flutter-sdk/bin/flutter run -d chrome --dart-define=enable_experiments=true
```

Comment:
Can we add instructions to the "# Development" section above for running DevTools with experiments? And including how to configure this flag in VS code would be nice.